README.md

```diff
@@ -20,6 +20,19 @@ to the [Charm Layers](https://jujucharms.com/docs/devel/developer-layers)
 documentation on [jujucharms.com](https://jujucharms.com/docs) for more 
 information.
 
+# States
+
+The following states are set by this layer:
+
+* `cuda.supported`
```

Comment:
Consider changing these state names to `docker-cuda.[supported|installed]`.

I fear they will interfere with [layer-nvidia-cuda](https://github.com/juju-solutions/layer-nvidia-cuda#states) states if a charm includes that layer in addition to this one.